starter-code/views/auth/boss.ejs

```diff
@@ -0,0 +1,47 @@
+<% if (checkBoss){%>
```

Comment:
This is causing a problem since checkBoss is not defined on this page. The better way to do it is to pass the user to the view when you render, and then check what "user.role" is.